Makefile

```diff
@@ -1,9 +1,21 @@
-make test_backend:
-	docker-compose run --rm backend python3 manage.py test && pylint apps
```

Comment:
@magul @jacol12345 I had to remove '&& pylint apps' as it crashed the container. The error message '/bin/sh: 1: pylint: not found'. I guess pylint should be added to the conainer first.

Reply:
Let's leave pylint to Travis for now. Later we should have a separate image for tests with all test requirements installed.

Reply:
Docker image should contain `pylint`, see:
* Dockerfile for backend image: https://github.com/CodeForPoznan/volontulo/blob/master/backend/Dockerfile#L29
* Requirements: https://github.com/CodeForPoznan/volontulo/blob/master/backend/requirements/dev.txt#L19

I agreee, that there should be separate `Makefile` task for linting but if can You test it again on fresh images from DockerHub @CorneeSean ?

Reply:
Ok, I believe I undestand what happened here - `&&` is already outside of docker container bash and `pylint apps` is executed on You host machine

can You test:
* `docker-compose run --rm backend pylint apps`
* `docker-compose run -rm backend python3 manage.py test && pylint apps`